cms/templates/partials/admissions-section.html

```diff
@@ -0,0 +1,61 @@
+{% load image_version_url wagtailcore_tags humanize %}
+<div class="container-fluid admissions-section">
+    <div class="row">
+        <img class="admissions-image" src="{% image_version_url page.admissions_image 'fill-1920x540' %}" alt="Admissions" />
+    </div>
+    <div class="row mb-4">
+        <div class="col-md-10 offset-md-2 col-sm-12 offset-sm-0 pt-5 bg-white">
+            <div class="container-fluid admissions-section">
+                <div class="row">
+                    <div class="col-md-4 col-sm-12 mb-2">
+                        <h1 class="section-heading">How to Apply</h1>
+                        <h2 class="admissions-heading mt-2"><a href="/admissions">Admissions</a></h2>
+                    </div>
+                    <div class="col-md-7 col-sm-12 admissions-detail">
+                        <div class="container-fluid">
+                            <div class="row">
+                                <div class="col">
+                                    {{ page.details|richtext }}
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col d-flex flex-row">
+                                    <div class="flex-shrink-0">
+                                        <a href="/admissions" class="btn btn-primary btn-apply">Apply Now</a>
+                                    </div>
+                                    <div class="notes d-flex flex-grow-1 ml-3">{{ page.notes }}</div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row pb-5 very-light-grey">
+        <div class="col-md-10 offset-md-1 col-sm-12 offset-sm-0">
+            <div class="container-fluid">
+              <div class="row text-med-dark-grey">
+                <div class="col-md-4 col-sm-12 mt-5 border-left border-dark d-flex flex-column">
+                    <h3 class="block-title">Format</h3>
+                    <h2 class="font-weight-bold pb-2">{{ page.bootcamp_format }}</h2>
```

Comment:
I'm not seeing a difference in my local app between 600 font weight (which is defined in our basic `h2` styles) and 700 font weight (which is set by this `font-weight-bold` class). I'm not sure we need that extra 100 font weight. Unless I'm wrong about that, can you either get rid of that class definition entirely, or if it's used elsewhere, change the class name to `font-700`?

Reply:
Removed